Libraries/LibJS/Runtime/Object.cpp

```diff
@@ -645,21 +645,21 @@ bool Object::put_by_index(u32 property_index, Value value)
         if (interpreter().exception())
             return {};
     }
-    return put_own_property_by_index(*this, property_index, value, default_attributes, PutOwnPropertyMode::Put);
+    return put_own_property_by_index(*this, property_index, value, default_attributes, PutOwnPropertyMode::Put, throw_exceptions);
 }
 
-bool Object::put(PropertyName property_name, Value value)
+bool Object::put(PropertyName property_name, Value value, bool throw_exceptions)
```

Comment:
Why is this change needed? Is it effectful? I don't see us branching on this value anywhere.

Reply:
Ahh, looks like the spec actually does a `[[DefineProperty]]` in `InternalizeJSONProperty` instead of a `[[Set]]`, so yeah this isn't needed.